test/test_decorators.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from src.decorators import log
+
+
+@log(filename="my_log.txt")
+def my_function(x: int, y: int) -> (int, float):
```

Comment:
При делении всегда возвращается float
```suggestion
def my_function(x: int, y: int) -> float:
```